Assets/Scripts/PlayerFall.cs

```diff
@@ -0,0 +1,69 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class PlayerFall : MonoBehaviour
+{
+    
+    private Rigidbody PlayerBod;
+    //speed at which the player is dragged down
+    public float thrust = 10.0f;
+    private GameObject Camera;
+    private GameObject Player;
+    //height from Player we want Camera to be
+    private float desiredCameraHeight = 5.0f;
+    private Vector3 direction;
+    private float timer = 0.0f;
+    private float delayuntilFastFall = .5f;
+    
+    //falling audio at this time overriden in BeginFall method
+    [SerializeField]
+    private AudioSource fallingAudio;
+    
+    //flag
+    public bool trapOpened {get; set;}
+    
+    void Start(){
+        
+        GameObject.Find("TrapDoor").transform.GetChild(0).transform.GetChild(0).GetComponent<TrapDoorOpen>().TrapOpened += BeginFall;

```

Comment:
Sweet mother of god! This is wrong. Don't do this. Burn these two lines plz.

Find a better way of doing this that is not dependent on the gameobject's name and children ordering...

Reply:
Lol this FindComponentsInChildren is a God method. Love it.